src/schema/schema.sql

```diff
@@ -0,0 +1,65 @@
+CREATE TABLE `seasons` (
```

Comment:
What do you think about calling this `series` to be consistent with the [taskmaster nomenclature](https://taskmaster.fandom.com/wiki/Series_1)?